### PR TITLE
Bug  1590743 - Include all webhook types when listing all

### PR DESCRIPTION
**Pull request by soltysh**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1590743
/assign @adambkaplan

### CODE REVIEW

File: pkg/oc/cli/startbuild/startbuild.go [hunk 1]
```diff
@@ -347,13 +347,24 @@ func (o *StartBuildOptions) Validate() error {
 	if len(o.FromBuild) != 0 && o.AsBinary {
 		// TODO: we should support this, it should be possible to clone a build to run again with new uploaded artifacts.
 		// Doing so requires introducing a new clonebinary endpoint.
-		return fmt.Errorf("Cannot use '--from-build' flag with binary builds")
+		return fmt.Errorf("cannot use '--from-build' flag with binary builds")
```
Comment (adambkaplan) [inline pkg/oc/cli/startbuild/startbuild.go]: Change in case is breaking `cmd` test.

Comment (soltysh) [reply pkg/oc/cli/startbuild/startbuild.go]: I've fixed the tests, errors should always be lowercase. ptal

File: test/cmd/builds.sh [hunk 2]
```diff
@@ -123,9 +123,9 @@ started="$(basename $(oc start-build -o=name ruby-sample-build-invalidtag))"
...
 frombuild="$(basename $(oc start-build -o=name --from-build="${started}"))"
 os::cmd::expect_success_and_text "oc describe build ${frombuild}" 'centos/ruby-22-centos7$'
-os::cmd::expect_failure_and_text "oc start-build ruby-sample-build-invalid-tag --from-dir=. --from-build=${started}" "Cannot use '--from-build' flag with binary builds"
-os::cmd::expect_failure_and_text "oc start-build ruby-sample-build-invalid-tag --from-file=. --from-build=${started}" "Cannot use '--from-build' flag with binary builds"
-os::cmd::expect_failure_and_text "oc start-build ruby-sample-build-invalid-tag --from-repo=. --from-build=${started}" "Cannot use '--from-build' flag with binary builds"
+os::cmd::expect_failure_and_text "oc start-build ruby-sample-build-invalid-tag --from-dir=. --from-build=${started}" "cannot use '--from-build' flag with binary builds"
+os::cmd::expect_failure_and_text "oc start-build ruby-sample-build-invalid-tag --from-file=. --from-build=${started}" "cannot use '--from-build' flag with binary builds"
+os::cmd::expect_failure_and_text "oc start-build ruby-sample-build-invalid-tag --from-repo=. --from-build=${started}" "cannot use '--from-build' flag with binary builds"
```
Comment (adambkaplan) [inline test/cmd/builds.sh]: @soltysh can we add tests here to make sure we're listing the web hooks correctly?

**Comment by soltysh** [comment]
@adambkaplan added the tests you've asked for, ptal

**Pull request merged**